CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+### Added
+## 6.0.0 (Unreleased)
```

Comment:
Can you changed this to `v6.0.0`, the auto-tagging script requires the 'v' when it parses the changelog.

Reply:
And put "v6.0.0" above "added"